### PR TITLE
Add support for `screen` command

### LINKED ISSUE #18: Support `screen` call
As this server-side implementation is used by the iOS/Android integrations, it would great if it could support the `screen` event.

A suggested path would be to track `screen` events as a `track` event with the screen name as part of the event name:

`screen('Home', args)` --> `track('Viewed Home Screen', args)`

This is how at least some of the iOS implementations work today that don't support native screen events, eg.:
https://github.com/segment-integrations/analytics-ios-integration-amplitude/blob/master/Pod/Classes/SEGAmplitudeIntegration.m#L116-L122

What do you think?

### PR DESCRIPTION
**Pull request by pfeiffer**
This commit closes #18.

This is my first Segment.io implementation, and I hope things are all right. Tests are passing. Let me know if anything should be changed.

### CODE REVIEW

**Comment by WesleyDRobinson** [comment]
👀  @sperand-io @hankim813 @tsholmes 
jira 🎫  ref: EPD-1938 or EPD-1917

**Comment by pfeiffer** [comment]
Ping @WesleyDRobinson

File: lib/mapper.js [hunk 1]
```diff
@@ -92,3 +92,24 @@ exports.page = function(msg){
...
+ */
+
+exports.screen = function(msg){
+  var props = traverse(msg.properties());
+  var name = `Viewed ${msg.name()} Screen`;
+  delete props.name;
+
+  return {
+    timestamp: time(msg.timestamp()),
+    type: 'track',
```
Comment (hankim813) [inline lib/mapper.js]: @pfeiffer is this what we want to set here? Not sure amplitude has a "screen" counter type for `type` here given that we are sending `'page'` for `.page()` calls.

Comment (pfeiffer) [reply lib/mapper.js]: You're right! Will fix and rebase.

**Comment by hankim813** [comment]
@pfeiffer I see that you removed the `type` entirely. Could you also link in this PR that amplitude docs you are using for reference 🙏  :) 

I'd like to make sure that we can remove the field or see if there is a `screen` type that Amplitude accepts? Thanks for your help!

**Comment by pfeiffer** [comment]
Well, this PR is for Customer.io, not Amplitude.

Customer.io doesn't have a special event for `screen`. Other implementations with the similar trait (eg. Segment+Amplitude iOS) simply sends `screen` events as a custom event with a special naming. This PR adds the same functionality to Customer.io server integration.

Relevant part of Customer.io integration: https://learn.customer.io/api/#apievent_add

Segment+Amplitude iOS integration: https://github.com/segment-integrations/analytics-ios-integration-amplitude/blob/master/Pod/Classes/SEGAmplitudeIntegration.m#L116-L122

**Comment by hankim813** [comment]
@pfeiffer hahah oops I meant C.io ... mondays haha

great! last thing -- could you push a release commit?

bump `package.json` minor version
update `HISTORY.md` following conventions
git commit -am 'Release x.x.x'

thanks!

**Comment by pfeiffer** [comment]
Sure, done.

**Comment by hankim813** [comment]
@ladanazita / @anoonan can you guys deploy this this week with @Peripheral1994 to shadow? Not sure why circle ci is not running though.

**Comment by ladanazita** [comment]
merged in #26

**Pull request closed without merging**